plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerConnectorTest.java

```diff
@@ -351,9 +352,9 @@ public void testPredicatePushdown()
             assertThat(query(joinPushdownEnabled, "SELECT c.name, n.name FROM customer c JOIN nation n ON c.custkey = n.nationkey WHERE n.name = 'POLAND'"))
                     .isFullyPushedDown();
 
-            // join on varchar columns is not pushed down
-            assertThat(query(joinPushdownEnabled, "SELECT c.name, n.name FROM customer c JOIN nation n ON c.address = n.name"))
-                    .joinIsNotFullyPushedDown();
+            // join on varchar columns
+            assertThat(query(joinPushdownEnabled, "SELECT n.name, n2.regionkey FROM nation n JOIN nation n2 ON n.name = n2.name"))
```

Comment:
Can you please provide a tests with join pushdown with predicate on case insensitive columns?

Reply:
Test added (testNoJoinPushdownOnCaseInsensitiveVarcharColumn)